File: day3/program2.py
n = int(input("number: "))

even = 0
odd = 0
while(n>0):
    rem = n%10
    if(rem%2==0):
        even+=1
    else:
        odd+=1
    n = n//10
print(f"even:{even}, odd: {odd}")

File: day3/program2.java
package day3;

public class program2 {

    public static void main(String[] args){


        int n = 709456;
        int even = 0;
        int odd = 0;

while(n>0){
    

    int rem = n%10;
    if(rem%2==0){
        even++;
    }else{
        odd++;
    }
    n = n/10;

}
System.out.print("even: "+even+"odd: "+odd);
        
    }
    
}
